src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleTable } from './PeopleTable/PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [wrongLoading, setWrongLoading] = useState(false);
```

Comment:
Consider renaming `wrongLoading` to something more descriptive like `hasError` or `isError` to better convey its purpose as an error state indicator.